src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -358,6 +358,17 @@ public function getVersion()
     }
 
     /**
+     * @VirtualProperty
+     * @SerializedName("subVersion")
+     *
+     * @return int
+     */
+    public function getSubVersion()
+    {
+        return $this->getFileVersion()->getSubVersion();
```

Comment:
The sub-version is just a counter which counts the number of times the format-options have been changed. This is needed for invalidating the browser cache, once a new crop has been defined. The "subversion" could be set per format-option. However to realize this the way we add the thumbnail-urls to the response would have to be refactored.